tests/e2e/testfiles/storage_csm_powerflex.yaml

```diff
@@ -104,6 +104,28 @@ spec:
         # Default value: false
         - name: X_CSI_HEALTH_MONITOR_ENABLED
           value: "false"
+        # X_CSI_APPROVE_SDC_ENABLED: Enables/Disable SDC approval
+        # Allowed values:
+        #    true: enable SDC approval
```

Comment:
Can we change this comment to be more descriptive?

    #  true: Driver will attempt to approve restricted SDC by GUID during setup
    #  false: Driver will not attempt to approve restricted SDC by GUID during setup

Reply:
oh I see this is a test file- nevermind